pkg/release/config/client.go

```diff
@@ -22,12 +22,19 @@ func endpoint(c api.Candidate) string {
 type Job struct {
 	Name                 string `json:"name"`
 	api.MetadataWithTest `json:",inline"`
+
+	AggregatedCount int `json:"aggregatedCount,omitempty"`
```

Comment:
```suggestion
	AggregatedCount int `json:"-"`
```
This field should be ignored in marshal/unmarshal, right?

Reply:
Well, we never marshal/unmarshal this struct. To be honest I added the tag because I was unmarshaling it while debugging.

Reply:
We did. `Job` is part of `Verify`.
https://github.com/openshift/ci-tools/blob/26d2a699900570e2eff559da8a8543d8b298b493/pkg/release/config/client.go#L73

Reply:
We did. `Job` is part of `Verify`.
https://github.com/openshift/ci-tools/blob/26d2a699900570e2eff559da8a8543d8b298b493/pkg/release/config/client.go#L73